spec.html

```diff
@@ -24768,7 +24768,7 @@ <h1>Object.prototype.valueOf ( )</h1>
         <emu-alg>
           1. Return ? ToObject(*this* value).
         </emu-alg>
-        <p>This function is the <dfn>%ObjProto_valueOf%</dfn> intrinsic object.</p>
+        <p>This function is the <dfn>%Object.prototype.valueOf%</dfn> intrinsic object.</p>
```

Comment:
please revert this; as long as `%ObjProto_valueOf%` and similar exists in the table, these prose comments referencing it should remain.

Reply:
I don't understand: why should other references change to new-style but not these?

Reply:
I mean prose that mentions them.

Reply:
Okay, but why?

Reply:
This entire line isn't needed unless `%ObjProto_valueOf%` exists, because it's already implied by being reachable from `%Object%`. As such, these lines should remain untouched, until a future PR removes both them and the legacy intrinsics.

Reply:
Ah! I had wondered about the tautology of `X.Y is the intrinsic object %X.Y%`.

Reply:
But wait, in f1b22ef, you changed a bunch of prose-y references from old-style to new-style. Is there something different about them, or was that just a mistake? And do you want me to revert them back to old-style too?

Reply:
I changed all of them except the ones that were in these bullet lists, defining the objects at their creation.

Reply:
What about line 24664 (`is the intrinsic object <dfn>%Object.prototype%</dfn>`)
and line 38776 (`is the intrinsic object <dfn>%AsyncFunction.prototype%</dfn>`)?

Reply:
oops, looks like i mistakenly changed those. sorry for the confusion